tests/Unit/Domain/Test_DomainHelpers.cpp

```diff
@@ -330,13 +330,16 @@ void test_wedge_map_generation_against_domain_helpers(
   const double x_coord_of_shell_center = 0.1;
   const bool use_half_wedges = true;
   const double aspect_ratio = 1.0;
-  const bool use_logarithmic_map = true;
+  std::vector<double> radial_partitioning{1., 1.5};
```

Comment:
again, technically not the some domain, but I guess the test doesn't care...

Reply:
yes, since these were not specific edge cases or anything I felt it would be nicer this way. Putting in the logarithmic values for the boundary as a double would probably just cause confusion as to why this number was chosen.